src/gamma.jl

```diff
@@ -412,7 +412,7 @@ function zeta(s::ComplexOrReal{Float64})
         end
         if absim > 12 # amplitude of sinpi(s/2) ≈ exp(imag(s)*π/2)
             # avoid overflow/underflow (issue #128)
-            lg = lgamma(1 - s)
+            lg = logabsgamma(1 - s)
```

Comment:
`s` can be complex, so shouldn't this be `loggamma(1-s)`? Can you check the result?

Reply:
All the other functions were written previously, assuming the `logabsgamma` functionality I guess. Hence I had kept them the same. Also, `logabsgamma` also has a function with arg as `x::Number`, so that actually catches this case.

Reply:
My point is that `logabsgamma(x::Complex)` will give a different result than the current `lgamma(x::Complex)`

Reply:
It works both with `loggamma` and `logabsgamma` as of now I checked.

Reply:
That's because `logabsgamma` is incorrectly defined for `Complex` arguments, see my comment below.

Reply:
oh my bad! Due to network issue that comment didn't appear earlier.